natlas-server/app/errors/handlers.py

```diff
@@ -1,24 +1,47 @@
-from flask import render_template
+from flask import request
 from app.errors import bp
 from app import db
 import elasticsearch
 import sentry_sdk
 
+from .http_error import HTTPError
+from .responses import get_response, get_supported_formats
+
+
+def build_response(err: HTTPError):
+    selected_format = request.accept_mimetypes.best_match(
+        get_supported_formats(), "application/json"
```

Comment:
Can we slap a `default="application/json"` so it's obvious what that parameter is?